CMakeLists.txt

```diff
@@ -415,6 +417,30 @@ endif()
 
 # build the lib itself
 if(BUILD_LIBRARY)
+    ExternalProject_Get_Property(xlsxio_project INSTALL_DIR)
```

Comment:
I tested it with just add_subdirectory(), and although it worked fine, it *really* polluted our option space with junk. it becomes very difficult to configure dfhack with ccmake since there are so many options